src/main/java/sirius/db/es/ElasticQuery.java

```diff
@@ -1421,13 +1431,13 @@ public Stream<E> streamBlockwise() {
     }
 
     private class ElasticScrollingSpliterator extends PullBasedSpliterator<E> {
-        private long lastScroll = 0;
-        private String scrollId = null;
         private final TaskContext taskContext = TaskContext.get();
+        private String pit = null;
+        private List<String> searchAfter = null;
 
         @Override
         public int characteristics() {
-            return Spliterator.NONNULL | Spliterator.IMMUTABLE;
+            return NONNULL | IMMUTABLE;
```

Comment:
we commonly don't use static imports

Reply:
This is not a static import, this is a reference to a superclass constant, which does not need to be further qualified (when it is not ambiguous)